lib/mutant.rb

```diff
@@ -204,7 +204,11 @@ class Config
       integration:       Integration::Null,
       isolation:         Mutant::Isolation::Fork,
       jobs:              Mutant.ci? ? CI_DEFAULT_PROCESSOR_COUNT : ::Parallel.processor_count,
+      kernel:            Kernel,
```

Comment:
No direct references to `Kernel` exist inside the mutant namespaces anymore, correct. But other IO globals are still referenced and need future iterations, this PR is only the first step.

I extracted this PR out of a bigger change, as other changes need refactorings to be applied first.